pages/index.tsx

```diff
@@ -8,19 +8,30 @@ import Deposit from '../components/deposit'
 import Borrow from '../components/borrow'
 import { Asset } from '../model/asset'
 import { fetchAssets } from '../api/asset'
+import { fetchMarkets } from '../api/market'
 
 export const getServerSideProps: GetServerSideProps<{
   assets: Asset[]
+  dates: string[]
 }> = async () => {
-  const assets = await fetchAssets()
+  const [assets, markets] = await Promise.all([fetchAssets(), fetchMarkets()])
   return {
-    props: { assets },
+    props: {
+      assets,
+      dates: markets
+        .map((market) => market.endTimestamp)
```

Comment:
Let's sort and uniq on `bigint` instead of `string`. Sorting `string` might behave unexpectedly. For instance, on the next century 99/06/31 -> 00/06/31 will sort the wrong way.

Something like
```
[...new Set(markets
    .map((market) => market.endTimestamp))]
    .sort()
    .map((timestamp) =>
          new Date(Number(timestamp) * 1000).toISOString().slice(0, 10),
        )
```

Reply:
@onemonster https://github.com/clober-dex/coupon.finance/pull/40/commits/31c3a5d9cb6f116acb6afac9dc159602bc594f16 Nice!